tools/packages.yaml

```diff
@@ -3879,6 +3879,11 @@ url: https://www.gnupg.org/ftp/gcrypt/libksba
 activity: none
 ---
 kind: url
+name: libluv
+url: https://github.com/luvit/luv
```

Comment:
The url here is supposed to indicate where the latest release can be downloaded.  For github projects, that is normally releases or tags.  The original intention of this file was to provide a manifest for package updates.

Reply:
So what should this be?

Reply:
`https://github.com/luvit/luv/releases`
Note: This is not the case for all github projects.  Sometimes there is only `/tags` or nothing at all.

Reply:
I made all the changes I could here...